inital commit

Ashe546 opened the pull request:
-initial commit

OLIPLICHE commented on src/module/html_component.js:
```diff
@@ -0,0 +1,38 @@
+//     <input type = 'text' class = 'add_todo' placeholder = 'add todo list here'>
+//     `;
+//   return element;
+// }
```
- [ ] Please remove these comment lines because they don't make your code clean.

OLIPLICHE commented on src/module/add_todo.js:
```diff
@@ -0,0 +1,17 @@
+  const addTodo = document.querySelector('.add_todo');
+  const todoObject = {
+    description: addTodo.value,
+    completed: 'checked',
```
- [ ] Please make sure the new task should have a property `completed` to set `false`. It is part of this project requirement: "By default, new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5)."
- [ ] Please  pay attention to this screenshort
![1](https://user-images.githubusercontent.com/63972670/173175940-afc4cd5b-1301-4602-a6a2-7632d68e9e01.PNG)

OLIPLICHE reviewed (changes requested):
Hi @Ashe546,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
Linter is green (the last commit have a green check mark (✅)) 👏
README file is descriptive 👏
Commit messages are meaningful 👏 👏
Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Ashe546 commented:
-Removed all hardcoded items from the tasks array.
-Created a new JavaScript file for the new functionality.
-Implemented a function for adding a new task (add a new element to the array).
-Implemented a function for deleting a task (remove an element from the array).
-Implemented a function for editing task descriptions.
-Deleting a task now updates all remaining items' indexes,

CollinsTatang reviewed (approved):
# Approved 👏  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
![image](https://user-images.githubusercontent.com/55596970/158098566-50966056-ce3d-4c4b-aac3-cce7a7d107fb.png)



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
